<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargerPage</class>
 <widget class="QWidget" name="ChargerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 20px, &quot;SimHei&quot;;	
}

QWidget
{
	background-color: black;
}
QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}

QPushButton:disabled
{
		border-radius:5px;
	font: 20px, &quot;SimHei&quot;;
    color: grey;
	background-color: rgb(221,221,221);
}</string>
  </property>
  <widget class="QLabel" name="label_batteryVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>244</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>蓄电池电压(V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryCurrent3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>283</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryCurrent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>283</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>蓄电池电流(A)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryCurrent1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>283</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_vehicleNo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>127</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>车号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryState4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>166</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryVoltage3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>244</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryCurrent4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>283</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryTemp4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>205</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryState">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>166</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>充电机状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_car4No">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>127</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryState1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>166</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryTemp1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>205</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car3No">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>127</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netVoltage2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>322</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryVoltage2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>244</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netVoltage4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>322</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryState2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>166</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryCurrent2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>283</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryTemp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>205</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>蓄电池温度(℃)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_netVoltage3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>322</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryState3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>166</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car2No">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>127</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryVoltage4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>244</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryTemp2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>205</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryTemp3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>205</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>322</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>电网电压(V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_batteryVoltage1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>244</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netVoltage1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>322</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_car1No">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>127</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netCurrent1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>400</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netCurrent2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>400</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netCurrent4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>400</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_supportVoltage3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>361</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_supportVoltage2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>361</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_supportVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>361</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>支撑电容电压(V)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_netCurrent3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>400</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_supportVoltage4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>361</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_supportVoltage1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>361</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_netCurrent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>电网侧电流(A)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cut1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>478</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>439</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>439</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>478</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>478</y>
     <width>209</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>439</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut1">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>478</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_cut3">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>478</y>
     <width>210</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>439</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 15px, &quot;SimHei&quot;;	</string>
   </property>
   <property name="text">
    <string>故障复位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reset4">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>439</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>440</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cut2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>480</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string>切除</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_EmergencyBroadcast">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>紧急广播</string>
   </property>
  </widget>
  <zorder>label_reset1</zorder>
  <zorder>label_cut1</zorder>
  <zorder>btn_cut1</zorder>
  <zorder>label_batteryVoltage</zorder>
  <zorder>label_batteryCurrent3</zorder>
  <zorder>label_batteryCurrent</zorder>
  <zorder>label_batteryCurrent1</zorder>
  <zorder>label_vehicleNo</zorder>
  <zorder>label_batteryState4</zorder>
  <zorder>label_batteryVoltage3</zorder>
  <zorder>label_batteryCurrent4</zorder>
  <zorder>label_batteryTemp4</zorder>
  <zorder>label_batteryState</zorder>
  <zorder>label_car4No</zorder>
  <zorder>label_batteryState1</zorder>
  <zorder>label_batteryTemp1</zorder>
  <zorder>label_car3No</zorder>
  <zorder>label_netVoltage2</zorder>
  <zorder>label_batteryVoltage2</zorder>
  <zorder>label_netVoltage4</zorder>
  <zorder>label_batteryState2</zorder>
  <zorder>label_batteryCurrent2</zorder>
  <zorder>label_batteryTemp</zorder>
  <zorder>label_netVoltage3</zorder>
  <zorder>label_batteryState3</zorder>
  <zorder>label_car2No</zorder>
  <zorder>label_batteryVoltage4</zorder>
  <zorder>label_batteryTemp2</zorder>
  <zorder>label_batteryTemp3</zorder>
  <zorder>label_netVoltage</zorder>
  <zorder>label_batteryVoltage1</zorder>
  <zorder>label_netVoltage1</zorder>
  <zorder>label_car1No</zorder>
  <zorder>label_netCurrent1</zorder>
  <zorder>label_netCurrent2</zorder>
  <zorder>label_netCurrent4</zorder>
  <zorder>label_supportVoltage3</zorder>
  <zorder>label_supportVoltage2</zorder>
  <zorder>label_supportVoltage</zorder>
  <zorder>label_netCurrent3</zorder>
  <zorder>label_supportVoltage4</zorder>
  <zorder>label_supportVoltage1</zorder>
  <zorder>label_netCurrent</zorder>
  <zorder>btn_reset1</zorder>
  <zorder>label_cut</zorder>
  <zorder>label_reset2</zorder>
  <zorder>label_cut4</zorder>
  <zorder>label_cut2</zorder>
  <zorder>label_reset3</zorder>
  <zorder>label_cut3</zorder>
  <zorder>label_reset</zorder>
  <zorder>label_reset4</zorder>
  <zorder>btn_reset2</zorder>
  <zorder>btn_EmergencyBroadcast</zorder>
  <zorder>btn_cut2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
